device/bgp/bgpupdatereplay.yaml

```diff
@@ -0,0 +1,31 @@
+components:
+  schemas:
+    Bgp.UpdateReplay:
+      description: >-
+        Ordered BGP Updates ( including both Advertise and Withdraws ) to be sent to the peer after the BGP session is established.
```

Comment:
"to be sent in the order given in the input"

Reply:
Done.